packages/atomic/src/pages/examples/folding.html

```diff
@@ -76,6 +86,365 @@
           <atomic-layout-section section="results">
             <atomic-smart-snippet></atomic-smart-snippet>
             <atomic-folded-result-list image-size="small">
+              <atomic-result-template must-match-sourcetype="Slack">
+                <template>
+                  <style>
+                    div.result-root.with-sections.display-list.density-normal atomic-result-section-visual {
+                      margin-right: 16px;
+                    }
+
+                    .slack-section-badge {
+                      align-items: center;
+                      display: flex;
+                      width: 100%;
+                    }
+
+                    .slack-title-icon {
+                      display: flex;
+                      width: 100%;
+                      margin-top: 8px;
+                    }
+
+                    .slack-badge::part(result-badge-element) {
+                      background-color: #86d8ff;
+                      color: white;
+                      border-radius: 32px;
+                      padding-top: 2px;
+                      padding-bottom: 2px;
+                    }
+
+                    ::part(children-root) {
+                      margin-left: 70px;
+                      border-left: thin solid #bcc3ca;
+                    }
+
+                    div.result-root atomic-result-image {
+                      height: 30px;
+                      width: 30px;
+                      margin-right: 16px;
+                      border-radius: 4px;
+                    }
+
+                    .slack-atomic-result-icon {
+                      flex-shrink: 0;
+                      height: 32px;
+                      width: 32px;
+                      background-size: cover;
+                      margin-right: 16px;
+                    }
+
+                    .slack-msg-result-section-title {
+                      display: flex;
+                      background-color: #f6f7f9;
+                      border: 1px solid #f6f7f9;
+                      border-radius: 8px;
+                      flex-grow: 1;
+                      padding: 8px 8px 0px 8px;
+                    }
+
+                    .slack-link {
+                      color: #1372ec;
+                      font-size: 12px;
```

Comment:
I would recommend that you use our font size variables here.

For instance: https://github.com/coveo/ui-kit/blob/fd0382e36f60be331f309feaf79a8380da41921b/packages/atomic/src/themes/coveo.css#L36